mindocr/models/det_fcenet.py

```diff
@@ -34,12 +34,11 @@ def fcenet_resnet50(pretrained=False, **kwargs):
         },
         "neck": {
             "name": 'FCEFPN',
-            "out_channels": 256,
+            "out_channel": 256,
```

Comment:
I think it is better to change to `out_channels` in `mindocr.models.necks.fpn.FCEFPN`.